documentation/docs/developers/connectors/tutorial.md

```diff
@@ -306,7 +306,24 @@ MARKET_CLASSES = {
     .
     .
     "new_market": NewMarket
-}	}
+}
+.
+.
+.
+  def _initialize_markets(self, market_names: List[Tuple[str, List[str]]]):
+    ...
+    ...
+       ...
+       elif market_name == "new_market":
+         new_market_api_key = global_config_map.get("coinbase_pro_api_key").value
```

Comment:
should be ```new_market_api_key = global_config_map.get("new_market_api_key").value```?

Reply:
I missed that, thanks for the correction!